src/blog/article/BlogArticlePage.tsx

```diff
@@ -26,77 +28,124 @@ const BlogArticlePage: React.FC<BlogArticlePageProps> = ({
       title,
       content: { json },
       author,
+      updatedAt,
     },
     otherContentfulBlogArticles: { edges },
   },
   location: { href },
-}) => (
-  <>
-    <RippedSection rips={{ bottom: { color: 'white' } }}>
-      <Image
-        fluid={getImageForBreakpoint(hero)}
-        style={{ width: '100%' }}
-        aria-hidden="true"
+}) => {
+  const orgName = 'Pride in London'
+  const firstParagraph = getFirstParagraph(json)
+  const image = `https:${hero.tablet.src}`
+  return (
+    <>
+      <Helmet
+        title={title}
+        siteUrl={href}
+        description={firstParagraph}
+        img={image}
       />
-    </RippedSection>
-    <Wrapper
-      display="flex"
-      justifyContent="center"
-      marginTop="xxl"
-      marginBottom="lg"
-    >
-      <Tag color={getCategoryColor(category)}>{category}</Tag>
-    </Wrapper>
-    <Wrapper
-      display="flex"
-      flexDirection="column"
-      alignItems="center"
-      justifyContent="center"
-      paddingX={{ default: 'lg', md: 'xxl' }}
-      position="relative"
-    >
-      <H2 as="h1" textAlign="center" maxWidth={842}>
-        {title}
-      </H2>
+      <ReactHelmet
+        script={[
```

Comment:
Try inspecting the rendered page... can you see the schema being included? It should exist in the `head` 🤔  might need to double-check how this is written!